Resources/Prototypes/Entities/Objects/Specific/chemistry.yml

```diff
@@ -243,3 +243,23 @@
     solutions:
       food:
         maxVol: 50
+
+- type: entity
+  name: pill canister
+  id: PillCanister
+  parent: BaseItem
+  description: Holds up to 12 pills.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Chemistry/pills_canister.rsi
+    state: pill_canister
+  - type: Item
+    sprite: Objects/Specific/Chemistry/pills_canister.rsi
+  - type: Storage
+    capacity: 60
```

Comment:
Holds up to 12 pills for a capacity of 60. Maybe pills should just be smaller in the first place? They currently just use the `BaseItem` size of 5. It should probably just be size 1.

Also, in principle the size of the canister should probably be at least as large as the capacity. But for whatever reason size is an integer, and you cant make pills smaller than 1, so that would probably lead to unreasonably large canisters. So it's probably fine to leave it at 1. The whitelist means this shouldn't be abusable.